TGC/events/Germany VS Hungary Events.txt

```diff
@@ -0,0 +1,672 @@
+                3308 = {  secede_province = GER }
+                634 = {  secede_province = GER }
+                636 = {  secede_province = GER }
+                638 = {  secede_province = GER }
+                635 = {  secede_province = GER }
+                637 = {  secede_province = GER }
+                3310 = {  secede_province = GER }
+                release = RUT
+                create_vassal = RUT
+                release = SLV
+                create_vassal = SLV
+diplomatic_influence = {
+	             who = RUT
+	             value = 400
+        
+                  }
+diplomatic_influence = {
+	             who = SLV
+	             value = 400
+        
+                  }
+        }  
+}
+
+##Freistaat Banat introduction##
+
+country_event = {
+	id = 2000000002
+	title = "EVTNAME2000000002"
+	desc = "EVTDESC2000000002"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000002"
+                  diplomatic_influence = {
+	             who = BAN
+	             value = 400
+        
+                  }
+BAN = { country_event = { id = 2000000003 days = 10 } }
+}
+}
+##Banat and Siebenbürgen Government set up##
+ 
+country_event = {
+	id = 2000000003
+	title = "EVTNAME2000000003"
+	desc = "EVTDESC2000000003"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000003"
+               government = dominion
+               primary_culture = south_german
+               add_accepted_culture = romanian
+               add_accepted_culture = north_german                         
+               remove_accepted_culture = hungarian
+        }
+
+}
+
+
+##Freistaat Siebenbürgen introduction##
+
+country_event = {
+	id = 2000000004
+	title = "EVTNAME2000000004"
+	desc = "EVTDESC2000000004"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000002"
+                  diplomatic_influence = {
+	             who = SIE
+	             value = 400
+                  }
+SIE = { country_event = { id = 2000000003 days = 10 } }
+
+}
+
+}
+##Hungarian client kingdom set up##
+
+country_event = {
+	id = 2000000005
+	title = "EVTNAME2000000005"
+	desc = "EVTDESC2000000005"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000005"
+                  diplomatic_influence = {
+	             who = HUN
+	             value = 400
+       
+                  }
+               HUN = { country_event = { id = 2000000006 days = 10 } }
+        }
+}
+
+##Habsburg King arrives##
+
+country_event = {
+	id = 2000000006
+	title = "EVTNAME2000000006"
+	desc = "EVTDESC2000000006"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000006"
+               add_accepted_culture = south_german 
+               political_reform = no_voting
+               government = absolute_monarchy

```

Comment:
Maybe have it be the same kind of monarchy as Germany has (so HMs Govt Germany would make a HMs Govt Hungary)?

Reply:
no, in my opinion this needs to be an absolute monarchy as the whole point of the restoration is to puppet hungary through making its monarch (who is part of germany and subject to the kaiser) dictate government. If the hungarians would be allowed to vote they probably wouldnt vote for the policy the german government has in mind.

Reply:
Makes sense, since the order would be difficult to keep with Hungarians demanding political rights and we could expand this to perhaps have Hungary become independent if they lose the monarchy or any party other than the pro-German reactionaries comes to power.

Reply:
I already have that done and also made an event to give transylvania its romanian primary culture back if it becomes independent. Although i based it around either becomeing independent or losing absolute monarchy instead of reactionary party being in power.